### DISCUSSION TITLE
Saleor 3 's rendering is much slower than 2.11

### DISCUSSION THREAD

**Discussion by sunny092020** [saleor/saleor#8125] [General]
I clone saleor 2.11 and it renders very fast.

But when I switch to version 3, it becomes much slower.

It needs more than 1s to serve a request (1.3 or 1.6s), v2.11 only needs less than 0.5s.

All tests are run on my local machine.

Can anybody tell me why - and how to fix this?

Thank you so much

**Comment by karolkielecki** [comment]
Please check how much RAM you have dedicated for Saleor. Saleor 3.0 is much faster than 2.11. If you run Saleor on an old device, you can experience some issues.

**Comment by sunny092020** [comment]
my pc is new with 16GB of RAM
I started the whole system using docker-compose up
the slow requests are like this

http://localhost:3000/_next/data/development/product/colored-parrot-cushion.json
http://localhost:3000/_next/data/development/category/homewares.json
http://localhost:3000/_next/data/development/product/light-speed-yellow-paint.json
...
they are all above 1.5 seconds latency 

how to turn off these JSON requests?

version 2.11 does not have these json requests
